lib/querystring.js

```diff
@@ -15,6 +15,7 @@ const QueryString = module.exports = {
   decode: parse
 };
 const Buffer = require('buffer').Buffer;
+const hexTable = require('internal/querystring').hexTable;
```

Comment:
I think we should be more consistent about how we import single items like this. Here we're using the pre-ES6 style, but in other places we are destructuring like `const { hexTable } = require('internal/querystring');`